best-practices/README.md

```diff
@@ -220,6 +220,11 @@ Browsers
 
 * Don't support versions of Internet Explorer before IE9.
 
+DNS
+---
+
+* Avoid using non-standard DNS entries (i.e. ALIAS, ANAME).
```

Comment:
@gylaz I think I'd prefer to stick with `Avoid` here. There are lots of valid DNS entries you could use, and listing them all seems more likely to cause problems than trying to remember the two common ones you should avoid.

I think adding some wording mentioning apex domains as @jferris suggested might help some.